app/services/bill_run.service.js

```diff
@@ -16,23 +16,30 @@ class BillRunService {
   * Note that the updated stats are _not_ saved back to the database; it is up to the caller to do this.
   *
   * @param {Object} transaction translator belonging to the bill run to find and assess
+  * @param {boolean} requestFromGenerateBillRun [false] If true then we are updating the summary within the 'generate bill run'
```

Comment:
I think this is the [JSDoc syntax for a default parameter](https://jsdoc.app/tags-param.html#optional-parameters-and-default-values).

```suggestion
  * @param {boolean} [requestFromGenerateBillRun] If `true` then we are updating the summary within the 'generate bill run'. Defaults to `false`
```

Certainly what I've been using but I could be wrong!